src/middlewared/middlewared/common/smart/smartctl.py

```diff
@@ -35,7 +35,7 @@
     if driver.startswith("ciss"):
         return [f"/dev/{driver}{controller_id}", "-d", f"cciss,{channel_no}"]
 
-    if driver.startswith("twa"):
+    if any(driver.startswith(s) for s in ("twa", "twe", "tws")):
```

Comment:
```
>>> 'twa'.startswith(("twa", "twe", "tws"))
True
>>> 'twe'.startswith(("twa", "twe", "tws"))
True
>>> 'tws'.startswith(("twa", "twe", "tws"))
True
>>> 'tww'.startswith(("twa", "twe", "tws"))
False

```

Reply:
![596](https://user-images.githubusercontent.com/669257/69811248-fb582280-11ed-11ea-9287-6352ff9f2374.jpg)

Reply:
You're gonna have to translate that now :P

Reply:
![LENIN-To-Learn 1  (1)](https://user-images.githubusercontent.com/669257/69811504-7c171e80-11ee-11ea-9aaf-33db49dda8c0.jpg)